src/types/offer/interfaces/offer.interfaces.ts

```diff
@@ -30,8 +33,8 @@ export interface Offer extends CommonEntityFields {
     | 'FAQ'
     | 'averageRating'
   >
-  subject: SubjectNameInterface
-  category: CategoryInterface
+  subject?: SubjectNameInterface
```

Comment:
I might be mistaken, but there is a task on the backend which states that both subject and category must be required.
Take a look here: [issue #77](https://github.com/orgs/WebUI-Project-Based-07-nr/projects/1/views/1?filterQuery=call&pane=issue&itemId=73388462)

Reply:
Yes, you are right, but right now I don't have it in models (backend). But I'll still rewrite it in the required case.
https://github.com/WebUI-Project-Based-07-nr/BackEnd/blob/develop/src/models/offer.js